### ISSUE TITLE
Invalid Jsonnet syntax causing jobs to fail. 

### ISSUE THREAD

**Issue by clbx** [EmbarkStudios/k8s-buildkite-plugin#64]
**Describe the bug**

[This line](https://github.com/EmbarkStudios/k8s-buildkite-plugin/blob/64e578d6547f7f3151f2756d7b036bb75df4f092/lib/job.jsonnet#L93) changed in #63 is failing in my jobs and is showing as invalid syntax in vscode. 

**To Reproduce**
1. Attempt to run a job with the most recent build.

**Expected behavior**
Not get a syntax error

**Screenshots**
```
Starting Kubernetes Job | 0s
-- | --
  | STATIC ERROR: /buildkite/plugins/github-com-greymatter-io-k8s-buildkite-plugin/lib/job.jsonnet:93:36: expected a comma before next array element.
  | 🚨 Error: The command exited with status 1
  | user command error: The plugin k8s command hook exited with status 1

```

![image](https://user-images.githubusercontent.com/15034517/218961368-94370b2b-58dd-4fc7-a5f6-c1165696182b.png)

I'm not sure what it does, but removing the line fixes the issue (though I suppose thats not the best fix)

**Comment by tgolsson** [comment]
Should be fixed in #66 now.

I'm working on a release; but I'm rewriting CI to also run itself with the current commit to ensure this kind of thing doesn't sneak through.

**Comment by tgolsson** [comment]
Fixed with #67 which also adds a first self-test.
